javascript/ql/src/Security/CWE-116/UnsafeHtmlExpansion.ql

```diff
@@ -0,0 +1,67 @@
+/**
+ * @name Unsafe expansion of shorthand HTML tag
```

Comment:
Isn't this called a self-closing tag in standard terminology? Could we switch to that instead of "shorthand"? (here and elsewhere)